src/pages/product-view.tsx

```diff
@@ -0,0 +1,19 @@
+import Footer from '../components/Footer'
```

Comment:
Please always import `react` on the first line. We are going to use its variable in the future.  
Ref: https://github.com/vercel/next.js/issues/12964#issuecomment-629642157